src/SimpleGraphs/simplegraph.jl

```diff
@@ -21,6 +21,26 @@ end
 
 eltype(x::SimpleGraph{T}) where {T} = T
 
+# SimpleGraph(g)
+"""
+    SimpleGraph{T}(g::AbstractGraph)
+
+Construct a `SimpleGraph{T}` from an `AbstractGraph`. 
+If `g` is a `SimpleDiGraph`, then every directed edge in `g` is 
+added as an undirected edge.
```

Comment:
```suggestion
Construct a `SimpleGraph{T}` from an `AbstractGraph`. 
If `g` is a directed, then every directed edge in `g` is 
added as an undirected edge.
```

Reply:
Also, how about we let the user specify what happens if `g` is directed? Then the signature could look something like
```julia
SimpleGraph(g::AbstractGraph; only_strong_edges::Bool = false)
```
or maybe there is a better name for `only_strong_edges` - I took the inspiration for the name from the concept of weak and strong components in directed graphs.